src/osx/cocoa/window.mm

```diff
@@ -450,8 +450,7 @@ long wxOSXTranslateCocoaKey( NSEvent* event, int eventType )
 
     wxevent.m_rawCode = [nsEvent keyCode];
     wxevent.m_rawFlags = modifiers;
-
-    wxevent.SetTimestamp( (int)([nsEvent timestamp] * 1000) ) ;
```

Comment:
Just to confirm, was removing the call to `SetTimestamp()` intentional?

Reply:
Excellent catch, my apologies -- This was entirely accidental.